cli/auth.go

```diff
@@ -28,9 +36,24 @@ var authCreateAdminToken = &cli.Command{
 
 		ctx := ReqContext(cctx)
 
-		// TODO: Probably tell the user how powerful this token is
+		if !cctx.IsSet("perm") {
+			return errors.New("--perm flag not set")
+		}
+
+		perm := cctx.String("perm")
+		idx := -1
+		for i, p := range api.AllPermissions {
+			if perm == p {
+				idx = i
+			}
+		}
+
+		if idx == -1 {
+			return fmt.Errorf("--perm flag has to be one of: %s", api.AllPermissions)
```

Comment:
shouldn't this be:
```suggestion
			return fmt.Errorf("--perm flag has to be one of: %v", api.AllPermissions)
```

Reply:
`%s` results in `[read write sign admin]`, `%v` is the same, just confirmed.